polylogue/cli/app.py

```
@@ -663,20 +873,30 @@ def _ensure(name: str, handler: Callable[[argparse.Namespace, CommandEnv], None]
     _ensure("prune", _dispatch_prune, "Remove legacy single-file outputs and attachments")
     _ensure("doctor", _dispatch_doctor, "Check local data directories for common issues")
     _ensure("status", _dispatch_status, "Show cached Drive info and recent runs")
+    _ensure("dashboards", _dispatch_dashboards, "Show provider dashboards")
+    _ensure("runs", lambda args, env: run_runs_cli(args, env), "List recent runs")
```

Ruff (0.15.9)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)